spec/dummy-app/spec/features/sign_in_spec.rb

```diff
@@ -162,4 +162,39 @@
       end
     end
   end
+
+  context 'older Recognisable sessions are still valid' do
```

Comment:
What does 'valid' mean for this test? Is it signing in using the older details? Newer ones?